projects/gis/src/app/app-config.service.ts

```diff
@@ -11,6 +11,9 @@ interface Components {
     readonly REGISTRATION: {
         readonly AVAILABLE: boolean;
     };
+    readonly MAP_INFO: {
```

Comment:
map info is very generic. maybe call it MAP_BBOX_RESOLUTION_OVERLAY?
or something more compact but more recognizable if you have an idea.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/720/commits/5f68e95e3c0c65f8f6ba5500915bc974b8d3fce8
https://github.com/geo-engine/geoengine-ui/pull/720/commits/10617c83eb85f4dc97e6169dd0eaa62681a39ae3